src/pyasm/web/html_wdg.py

```diff
@@ -851,7 +855,24 @@ def add_update_expression(my, expr):
         } )
 
 
+    def get_timezone_value(cls, value):
```

Comment:
Would it be useful to pass in a format?  So different widgets can use different formats?